bcda/main.go

```diff
@@ -466,16 +468,20 @@ func archiveExpiring(hrThreshold int) {
 			err = os.Rename(jobDir, expDir)
 			if err != nil {
 				log.Error(err)
+				lastJobError = err
 				continue
```

Comment:
If we're going to fail on an error, I think we should just fail and not continue. Otherwise, the status will be updated on the job when it really shouldn't be

Reply:
I could be misunderstanding the flow control here, but I think breaking this loop with a `continue` will skip the portion of the code that updates job status.

Reply:
@DeirdreHolub The `continue` stems from [Karl's comments](https://github.com/CMSgov/bcda-app/pull/66#discussion_r232318503) in the original PR.

I don't think the status will get updated on this job if the file manipulation fails, because `continue` simply continues with the next iteration in the `for` loop instead of executing the rest of the code in the block.  In this case, if the file manipulation fails, the job status will not be updated.